README.md

```diff
@@ -123,14 +137,59 @@ async function run() {
 
   // Example: delete a brand
   await courier.deleteBrand("<BRAND_ID>");
+
+  // Example: get all lists
+  const { paging, items } = await courier.lists.list({
+    cursor: "<CURSOR>" // optional
+  });
+  console.log(items);
+
+  // Example: get a specific list
+  const list = await courier.lists.get("<LIST_ID>");
+  console.log(list);
+
+  // Example: create or replace a list
+  const replacedList = await courier.lists.put("<LIST_ID>", {
+    name: "My New List"
+  });
+  console.log(replacedList);
+
+  // Example: delete a list
+  await courier.lists.delete("<LIST_ID>");
+
+  // Example: restore a list
+  await courier.lists.restore("<LIST_ID>");
+
+  // Example: get a list's subscriptions
+  const { paging, items } = await courier.lists.getSubscriptions("<LIST_ID>");
+  console.log(items);
+
+  // Example: subscribe many recipients to a list
+  await courier.lists.bulkSubscribe("<LIST_ID>", [
```

Comment:
might want to note this method will automatically create a `list` if one does not already exist

Reply:
I think I can add "to a new or existing list"